### ISSUE TITLE
Can not capture https web

### ISSUE THREAD

**Issue by JimmyHwang** [juliangruber/url-to-screenshot#8]
I tried put https url for capture, but it is doesn't work, http is work well.

**Comment by juliangruber** [comment]
Is there an error?

**Comment by JimmyHwang** [comment]
> Is there an error?

yes, http://www.google.com is work, but https not, I think it is problem, currently many sites it need https

**Comment by juliangruber** [comment]
What is the error message you are seeing when it fails?

**Comment by juliangruber** [comment]
Maybe check whether the system you are running this from has its time set correctly? That's one common source of https errors

**Comment by JimmyHwang** [comment]
> Maybe check whether the system you are running this from has its time set correctly? That's one common source of https errors

Yes, the system time is correct "Mon Dec 28 23:41:49 CST 2020", I also test another similar package, the "capture-screenshot-phantomjs" is work well for http and https both in my system.
My system is Ubuntu 16.04, nodejs v12.19.1
I also test it on Ubuntu 18.04, npm install url-to-screenshot, it is same issue

**Comment by juliangruber** [comment]
When it doesn't work, how does it not work? Is there an error message you're seeing?

**Comment by JimmyHwang** [comment]
> When it doesn't work, how does it not work? Is there an error message you're seeing?

I don't see any error message but had some garbage code in example.png when url is https.

**Comment by juliangruber** [comment]
Can you share the result?

**Comment by JimmyHwang** [comment]
I compress sample code & output image file, screen-shot in a 7z, please check this.
[Link](https://www.emagsoft.com/ScreenCapture/20201229.7z)

**Comment by juliangruber** [comment]
I can reproduce, wow!

It might be, that Phantomjs is outdated. Sorry, I don't have time to work on a fix right now
